source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -59,6 +59,15 @@ and their usage </reference/operator/update-field/>`.
 The update operators apply only to the fields associated with them in your
 update document.
 
+.. note:: Aggregation Pipelines in Update Operations
+
+   If you are using MongoDB Version 4.2 or later, you can use aggregation
+   pipelines made up of a subset of aggregation stages in update operations. For
+   more information, including what aggregation stages MongoDB supports in
+   aggregation pipelines used in update operations, see this tutorial on building
```

Comment:
```suggestion
   aggregation pipelines used in update operations, see our tutorial on building
```
Nit: I like to know who's content/material a link belongs to before considering to view it.

Reply:
Thanks for the note. I agree that knowing who is hosting a link or tutorial is helpful information for readers. Committing suggested change.

Perhaps this is something we could codify in a team PR review? (I.E. use "our" when mentioning content produced by MongoDB)